docs/reference/3-admin-api/installation/admin-api-2x-for-odsapi-7x/admin-api-2x-iis-installation-manual.md

```diff
@@ -0,0 +1,416 @@
+    installing the .NET Hosting Bundle, it may be necessary to restart the
+    computer for the changes to take effect.
+
+Installation Steps
+
+### **Step 1. Create Admin API Directory**
+
+Create a directory to hold all of the Admin API source files. In this example,
+we'll use a directory on the following path: "C:\\Ed-Fi\\AdminAPI".
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/Empty%20Installation%20Folder.png)
+
+### **Step 2. Rename and Unzip Admin API Source Files**
+
+Download and rename the linked Nuget Package (.npkg) to .zip
+
+Unzip the contents into the folder created in Step 1.
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/Populated%20Installation%20Folder.png)
+
+![Screenshot](https://odsassets.blob.core.windows.net/public/docs.ed-fi.org/reference/3-admin-api/img/installation-v2/image2024-5-7_13-7-50.png)
+
+There will be two folders. AdminApi folder will have binaries. Installer folder
+contains PowerShell scripts required for installation.
+
+### **Step 3. Ensure SQL Server Login Exists**
+
+In SQL Server Management Studio, make sure that you have a login for the
+database server. Guidance on how to set this up is explained in this [blog
+post](https://blogs.msdn.microsoft.com/ericparvin/2015/04/14/how-to-add-the-applicationpoolidentity-to-a-sql-server-login/).
+A few basics:
+
+* You can choose either Windows authentication or SQL Server authentication
+    here.
+* If you choose SQL Server authentication, make sure that if you have "User
+    must change password at next login" checked. This means you must connect to
+    SSMS with those credentials to reset the password. Otherwise, the app pools
+    won't be able to connect.
+* On the Server Roles page, ensure the "dbcreator" checkbox is checked since
+    Entity Framework will create the database when the application is launched.
+    You must do this when using either Windows or SQL Server authentication.
+
+Once you have confirmed a proper SQL Server login exists, continue to the next
+step.
+
+### **Step 4. Configure App/Web Configuration Files**
+
+You will need to manually edit connection strings, authorization settings, and
+keys in "AdminApi\\appsettings.json". Some values to note:
+
+* Authentication Settings
+  * `Authentication:SigningKey`  must be a Base64-encoded 256-bit string.
+        The following script demonstrates how to generate it, but you can use
+        another preferred method:
+
+        **generate\_signingkey.ps1** Expand source
+
+        ```
+        # SPDX-License-Identifier: Apache-2.0
+        # Licensed to the Ed-Fi Alliance under one or more agreements.
+        # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+        # See the LICENSE and NOTICES files in the project root for more information.
+
```

Comment:
Suggestion to remove the license header. Not really needed here, and it forces the reader take more time to get to the good stuff below it.

```suggestion
```